src/Components/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,47 @@
+import styles from "./SideBar.module.scss";
+
+const SideBar = () => {
+  const tags = [
+    "Kotlin",
+    "Vue Js",
+    "Android",
+    "ReactJs",
+    "Node",
+    "Placements",
+    "Embedded Structures",
+    "Data Structures",
+    "react",
+    "vueJs",
+    "Kotlin",
+    "pnpm",
+    "Vite",
+    "scss",
+  ];
+
+  const tagspan = tags.map((tag) => <span>{tag}</span>);
+
+  return (
+    <div className={styles.mainBox}>
+      <div className={styles.box1}>
+        <h4>Search by tags</h4>
+        {tagspan}
+      </div>
+
+      <div className={styles.box2}>
+        <h4>Bookmarks</h4>
+        <p>What is ReactJS and how can it be used int development</p>
```

Comment:
Convert these bookmarks into links, as well as using a map to iterate over an array of bookmarks. Also, just take the first 5 of them from the array (keep more than 5 elements in the array to check).

This is done as mostly your backend data may contain all of the bookmarks.

Also side tip: For dummy content, you can use [Lorem Ipsum](https://www.lipsum.com/) which is a standard for dummy text.